x-pack/plugins/integration_assistant/public/services/telemetry/types.ts

```diff
@@ -26,7 +26,7 @@ interface IntegrationAssistantOpenData {
   sessionId: string;
 }
 
-interface IntegrationAssistantStepCompleteData {
+interface IntegrationAssistantStepReadyToCompleteata {
```

Comment:
small typo here `IntegrationAssistantStepReadyToCompleteData`

Reply:
Thanks! Fixed it.